test/e2e/testdata/oidc-keycloak.yaml

```diff
@@ -72,11 +98,11 @@ spec:
     spec:
       initContainers:
         - name: wait-for-keycloak
-          image: busybox:stable
-          command: ["sh", "-c", "until nc -v -z -w3 keycloak 80; do sleep 2; done"]
+          image: curlimages/curl:latest
+          command: ["sh", "-c", "until curl -s -o /dev/null -w '%{http_code}' http://keycloak:80; do sleep 2; done"]
```

Comment:
Using curl instead of nc to verify that keycload is ready. This change fixed some flaky OIDC tests.